content/backend/graphql-js/3-a-simple-mutation.md

```diff
@@ -7,11 +7,11 @@ answers: ["It carries the return value of the previous resolver execution level"
 correctAnswer: 1
 ---
 
-In this section, you'll learn how to add a mutation to the GraphQL API. This mutation will allow to _post_ new links to the server.
+In this section, you'll learn how to add a mutation to the GraphQL API allowing you to _post_ new links to the server.
 
 ### Extending the schema definition
 
-Like before, you need to start by adding the new operation to your GraphQL schema definition.
+Start by adding the Mutation operation to `typeDefs` which is your GraphQL schema definition.
```

Comment:
I'd go with: `Start by adding new operation to your GraphQL schema definition.`, since the next line specifies the `typeDefs`